cmd/zetae2e/local/local.go

```diff
@@ -318,6 +325,52 @@ func localE2ETest(cmd *cobra.Command, _ []string) {
 
 	logger.Print("✅ e2e tests completed in %s", time.Since(testStartTime).String())
 
+	if testMigration {
```

Comment:
Would be cleaner to move this part in a sub function

Reply:
be5054dbfab44469ac21f1807b3ac34ae0883f74